packages/editor/src/components/panel-color-settings/test/__snapshots__/index.js.snap

```diff
@@ -1,35 +1,74 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`PanelColorSettings matches the snapshot 1`] = `
-<PanelBody
-  className="editor-panel-color-settings"
-  title={
-    <span
-      className="editor-panel-color-settings__panel-title"
-    >
-      Test Title
-      <ColorIndicator
-        aria-label="(border color: #000)"
-        colorValue="#000"
-      />
-      <ColorIndicator
-        aria-label="(background color: #111)"
-        colorValue="#111"
-      />
-    </span>
+<Component
+  colorSettings={
+    Array [
+      Object {
+        "label": "border color",
+        "onChange": [Function],
+        "value": "#000",
+      },
+      Object {
+        "label": "background color",
+        "onChange": [Function],
+        "value": "#111",
+      },
+    ]
   }
->
-  <WithColorContext(ColorPaletteControl)
-    key="0"
-    label="border color"
-    onChange={[Function]}
-    value="#000"
-  />
-  <WithColorContext(ColorPaletteControl)
-    key="1"
-    label="background color"
-    onChange={[Function]}
-    value="#111"
-  />
-</PanelBody>
+  colors={Array []}
+  title="Test Title"
+/>
+`;
+
+exports[`PanelColorSettings should render a color panel if at least one setting specifies some colors to choose 1`] = `
+<Component
```

Comment:
I noticed that the component used in the test is wrapped with `ifCondition`, which caused the shallow render to not render the children of PanelColorSettings (making the tests a bit obsolete). I found that you can use wrapper.dive() in enzyme to render the children, so I've pushed a commit with some additional snapshots.

Reply:
Thank you!